sylvia/tests/generics.rs

```diff
@@ -29,16 +34,30 @@ pub struct ExternalMsg;
 impl cosmwasm_std::CustomMsg for ExternalMsg {}
 impl sylvia::types::CustomMsg for ExternalMsg {}
 
+#[cw_serde]
+pub struct ExternalQuery;
+impl cosmwasm_std::CustomQuery for ExternalQuery {}
+
 #[cfg(test)]
 mod tests {
-    use cosmwasm_std::{CosmosMsg, Empty};
+    use cosmwasm_std::{testing::mock_dependencies, Addr, CosmosMsg, Empty, QuerierWrapper};
 
-    use crate::ExternalMsg;
+    use crate::{cw1::Querier, ExternalMsg, ExternalQuery};
 
     #[test]
     fn construct_messages() {
-        let _ = crate::cw1::QueryMsg::query(ExternalMsg {});
+        let contract = Addr::unchecked("contract");
+
+        let _ = crate::cw1::QueryMsg::<_, Empty>::some_query(ExternalMsg {});
         let _ = crate::cw1::ExecMsg::execute(vec![CosmosMsg::Custom(ExternalMsg {})]);
         let _ = crate::cw1::ExecMsg::execute(vec![CosmosMsg::Custom(Empty {})]);
+
+        // Generic Querier
+        let deps = mock_dependencies();
+        let querier: QuerierWrapper<'_, ExternalQuery> = QuerierWrapper::new(&deps.querier);
```

Comment:
You can omit the lifetime generics if they are to be elided:
```suggestion
        let querier: QuerierWrapper<ExternalQuery> = QuerierWrapper::new(&deps.querier);
```